src/ModelContextProtocol/Shared/McpSession.cs

```diff
@@ -16,6 +18,13 @@ namespace ModelContextProtocol.Shared;
 /// </summary>
 internal sealed class McpSession : IDisposable
 {
+    private static readonly ActivitySource s_activitySource = new("ModelContextProtocol");
```

Comment:
Do we want to use different activity source names for server and client sessions? Is there anything outside of the session, like connecting the client transport that we'd want to track with this ActivitySource?

Reply:
I don't think we need different names for the source, that is covered by the name of the actual activity. Having multiple sources just makes it more complicated for the user. IIRC we use `Microsoft.AspNetCore` for all of ASP.NET

Reply:
Yes, but Kestrel and HttpClient don't use the same source names, do they? I think it will be fairly common to have both client and server metrics in the same process, and while the client and servers are more peer-like than HTTP, they're still pretty different.

Reply:
I doubt people are going to be implementing both servers and clients in the same process, *and* want to only collect telemetry for one side of the communication.